spyder/plugins/editor/widgets/editor.py

```diff
@@ -621,7 +621,7 @@ def __init__(self, parent, actions):
         self.analysis_timer.timeout.connect(self.analyze_script)
 
         # Update filename label
-        self.editor_focus_changed.connect(self.update_fname_label)
+        self.editor_focus_changed.connect(lambda x, y: self.update_fname_label)

```

Comment:
```suggestion
        self.editor_focus_changed.connect(lambda x, y: self.update_fname_label())
```

I think this is required to call the corresponding method.